### ISSUE TITLE
Add feature to pick multiple colors

### ISSUE THREAD

**Issue by rydmike** [rydmike/flex_color_picker#34]
In some cases it might be useful to be able to pick multiple colors in one go and return them, maybe even with optional usage tags.

**Comment by throwyn** [comment]
Agree, Im actually searching for library that allows colors multiselect, but cannot find. I thought there is option in this library but this issue dispel doubts :(

**Comment by rydmike** [comment]
FlexColorPicker actually already does a sort of multi color init and return. The list of previous used colors, can be initialized, and it is returned too (so it can be saved and returned between app runs if so desired). The same mechanism is doable to also use for setting (or letting them be null) initial colors in a multi selection list of color boxes. Then you could in the picker select which one your are picking a color for at the moment, and return them all when done.

I think one could even allow a list of custom widgets to use as customizable labels for the colors being picked, plus a config if those label widgets are left, right, over, under or even inside the selected color box that one is picking the color for.
